release/supported-dxp-versions.txt

```diff
@@ -2,4 +2,4 @@
 7.3
 2023.q3
 2023.q4
-2024.q1
\ No newline at end of file
+2024.q1
```

Comment:
Hi @Wykenz, can you please give me more details how this commit fixes the problem?

From what I could see, it just removes a content and adds it back using a new line.

Thanks

Reply:
Hello @natocesarrego, as long as I saw the ` _promote_product_version` function's loop is reading a text file but it did not run on the last line so I added the new line for the end and this way the promotion is set to true for 2024.q1 versions

Reply:
> function's loop is reading a text file but it did not run on the last line so I added the new line for the end
Did you mean that just adding a new line to the end of the file fixed the problem, did I get it right?

If so, is there a way to make the code inside the function's loop more robust to allow it do the promotions without a new line? What do you think?

Reply:
Yes adding the new line fixed it, and I don't think there is an easy fix for changing it in the loop. I found this on google :
> "the read command fails when the input is not terminated with a newline. Since the newline character is missing at the end of your file, the read fails, so the last iteration of the while loop is skipped."
and as I see in other txt files in this repo the end of file is an empty new line